src/data/icons.json

```diff
@@ -15,7 +15,7 @@
   },
   "icons": [
     {
-      "name": "error",
+      "name": "error-stroke",
```

Comment:
Name changes are not backward compatible. Do you know if these are necessary? This will require a BREAKING CHANGE and a MAJOR release of the npm.

Reply:
I would argue that the name changes are necessary. These were named the way they are as there was no stroke or filled counterpart when they were first added in. Every other Icon with both a stroke or filled counterparts are named accordingly.